### PR TITLE
Document Load Event

### PR DESCRIPTION
**Pull request by Ashod**
- wsd: handle on-document-load events only once

### CODE REVIEW

File: net/HttpRequest.hpp [hunk 1]
```diff
@@ -1397,7 +1397,7 @@ class Session final : public ProtocolHandlerInterface
 
         std::shared_ptr<StreamSocket> socket = _socket.lock();
         if (socket)
-            socket->dumpState(os);
+            os << indent << "socket: #" << socket->getFD();
```
Comment (caolanm) [inline net/HttpRequest.hpp:1400]: Should we ditch the workaround of https://github.com/CollaboraOnline/online/commit/7769161afb19f18c484f90931bce0fc00c041dcc as a follow up?

**Review by caolanm — APPROVED** [review]

**Comment by caolanm** [comment]
unit-wopi-lock seems to be repeatedly failing here, which is suspicious

File: net/HttpRequest.hpp (hunk 1 above)
Comment (Ashod) [reply net/HttpRequest.hpp:1400]: Yes. I didn't remove it just yet until we all agree it's safe.

**Comment by Ashod** [comment]
> I thought we merged the loading: stuff already; perhaps I was confused =)

@mmeeks, yes, that was in the tests, this is in wsd. Inefficiencies and repeated "initialization" that I didn't realize we were doing. Now improved.

**Pull request merged**